conf/script/build-ha-io

```diff
@@ -621,8 +621,10 @@ motr_systemd_update() {
     log "${FUNCNAME[0]}: Adding Motr to Pacemaker..."
 
     cmd='
-    sudo sed "s/TimeoutStopSec=.*/TimeoutStopSec=5sec/"
-        -i /usr/lib/systemd/system/m0d@.service &&
+    sudo sed -e "s/TimeoutStopSec=.*/TimeoutStopSec=5sec/"
```

Comment:
If we are killing the process anyway then I think we don't need the timeout change. And timeout can be specified in m0d@.service systemd unit file itself.

Reply:
Already timeout of 5min is there in m0d@service, let's keep 5 sec here.
In future  if we want to remove pkill, we will be aware of it.
May be let's add a comment here.
Also is such changes being applied during updates ?

Reply:
Yeah I think, let's add a comment saying
"Killing m0d to reduce stop time in case of failover, thus overriding the systemd timeout."
>Also is such changes being applied during updates ?

Yes, they are applied during update and restore too,
```
 [motr_systemd_update]='bootstrap_update_restore'
```

Reply:
@VinothVijayakumar-dev : Can you add the comment here ?